StortSpelProjekt/Game/src/Gamefiles/Components/MeleeComponent.cpp

```diff
@@ -93,7 +90,7 @@ void component::MeleeComponent::Update(double dt)
 
 	float3 position = parentTransform->GetPositionFloat3();
 	float3 forwardVector = parentTransform->GetForwardFloat3();
-	float distanceFromPlayer = m_HalfSize.z + (modelDim.z * parentTransform->GetScale().z / 2.0f);
+	float distanceFromPlayer = m_HalfSize.z + (modelDim.z * parentTransform->GetScale().z);
```

Comment:
This makes the bounding box appear at different distances from the player depending on the level of the upgrade. The following code should be better:
`float distanceFromPlayer = (m_HalfSize.z * m_ZScale) + (modelDim.z * parentTransform->GetScale().z / 2.0f);`